src/utils/computeReClammInitAmounts.ts

```diff
@@ -0,0 +1,95 @@
+import { computeInitialBalanceRatio } from '@balancer-labs/balancer-maths';
+
+import { InputAmount } from '@/types';
+import { MinimalTokenWithRate } from '@/data/types';
+import { getSortedTokens } from '@/entities/utils/getSortedTokens';
+import { Token } from '@/entities/token';
+import { TokenAmount } from '@/entities/tokenAmount';
+import { TokenAmountWithRate } from '@/entities/tokenAmountWithRate';
+import { isSameAddress } from './helpers';
+import { MathSol } from './math';
+
+export type ComputeReClammInitAmountInput = {
+    chainId: number;
+    tokens: MinimalTokenWithRate[];
+    referenceAmountIn: InputAmount;
+    initialMinPrice: bigint;
+    initialMaxPrice: bigint;
+    initialTargetPrice: bigint;
+};
+
+export async function computeReClammInitAmounts({
```

Comment:
What is the reason this is async? I don't think it needs to be.

Reply:
nice catch! ya didn't need to be but will need after we start making `computeInitialBalances` fetch to pool contract